planner/core/planbuilder.go

```diff
@@ -1778,8 +1772,14 @@ func (b *PlanBuilder) buildValuesListOfInsert(ctx context.Context, insert *ast.I
 		for j, valueItem := range valuesItem {
 			var expr expression.Expression
 			var err error
+			var generatedColumnWithDefaultExpr bool
+			col := affectedValuesCols[j]
 			switch x := valueItem.(type) {
 			case *ast.DefaultExpr:
+				if col.IsGenerated() {
```

Comment:
Need to check if the DefaultExpr is only `default`.
```
mysql> insert into t value (default,  default(a));
Query OK, 1 row affected (0.01 sec)
```
However, in MySQL:
```
mysql> insert into t value (default,  default(a));
ERROR 3773 (HY000): DEFAULT function cannot be used with default value expressions
```

Reply:
@wjhuang2016 Comment Addressed